lib/pf/node.pm

```diff
@@ -822,7 +822,8 @@ sub node_register {
     if ( !pf::person::person_exist($pid) ) {
         $logger->info("creating person $pid because it doesn't exist");
         pf::person::person_add($pid);
-        pf::lookup::person::lookup_person($pid);
+        pf::lookup::person::lookup_person($pid,$info{'source'});
```

Comment:
As fabrice said, there is another place where we call lookup_person.

lib/pf/vlan.pm:                    pf::lookup::person::lookup_person($user_name);